### PR TITLE
Update quick-start.md

### PR DESCRIPTION
**Pull request by Yerion**
Added windows fix

### CODE REVIEW

File: _content/applications/mqtt/quick-start.md [hunk 1]
```diff
@@ -105,6 +109,11 @@ Otherwise, you'll have to use `payload_raw` and send a base64 encoded array of b
     ```bash
     mosquitto_pub -h <Region>.thethings.network -t '<AppID>/devices/<DevID>/down' -u '<AppID>' -P '<AppKey>' -m '{"payload_fields":{"led":true}}'
     ```
+    When using Windows, extra quotation marks have to be added to make the above code work:
+    
+    ```bash
+    mosquitto_pub -h <Region>.thethings.network -t "<AppID>/devices/<DevID>/down" -u "<AppID>" -P "<AppKey>" -m "{""payload_fields"":{""led"":true}}"
+
```
Comment (romeovs) [inline _content/applications/mqtt/quick-start.md]: Missing a closing ` ``` ` here.

**Review by romeovs — CHANGES_REQUESTED** [review]
I've cleaned up some whitespace.

Fix the one issue and it's good to go!

File: _content/applications/mqtt/quick-start.md (hunk 1 above)
Comment (Yerion) [reply _content/applications/mqtt/quick-start.md]: oke will do.

**Review by romeovs — APPROVED** [review]

**Comment by romeovs** [comment]
Your first PR @Yerion! Some comments:

`Update quick-start.md` is pretty hard to parse as a commit message, because it says nothing about what you did or why you did it. A good commit message chain here would be:
```
Add windows instructions for MQTT
Redacted whitespace
Fix missing Markdown hedge
```

**Comment by romeovs** [comment]
On another note. I see you have write access to this repository. No need to create PR's from your own repository in that case!

You can just
```
git clone https://github.com/TheThingsNetwork/docs
cd docs
git checkout -b fix/my-new-branch-with-a-better-name
# edit, commit, edit, commit, ...
git push 
```
and create a PR from that new branch within the repository. That makes it easier for other people to cooperate!

**Comment by romeovs** [comment]
@Yerion I've assigned you so you can merge!
